adam-core/src/main/scala/org/bdgenomics/adam/formats/avro/SingleReadBucketUtils.scala

```diff
@@ -0,0 +1,37 @@
+package org.bdgenomics.adam.formats.avro
+
+import org.bdgenomics.formats.avro.{ AlignmentRecord, Fragment, SingleReadBucket }
+
+import scala.collection.JavaConversions._
+
+object SingleReadBucketUtils {
```

Comment:
What do you think of the java convention of using the plural noun for the utility classname?  E.g. ```SingleReadBuckets```